frontend/templates/index.html

```diff
@@ -68,6 +68,11 @@ <h3>Preview</h3>
                 <button class="btn btn-primary mt-2" v-on:click="accept">Accept</button>
                 <button type="button" class="btn btn-danger mt-2" v-on:click="reject">Reject</button>
                 <button type="button" class="btn btn-info mt-2" v-on:click="skip">Skip</button>
+                <div class="form-group form-check mt-2">
+                  <input type="checkbox" class="form-check-input" id="comment" v-on:click="toggleCommentBox"/>

```

Comment:
instead of toggling, let us look at the value of check-box and decide based on it whether to show or hide the comment.

https://vuejs.org/v2/guide/forms.html#Checkbox